### ISSUE TITLE
TabBarView is not accessible

### ISSUE THREAD

**Issue by goderbauer** [flutter/flutter#12383]
`TabBarView` is not exposing any a11y elements of its children making its content unaccessible.

**Comment by goderbauer** [comment]
The underlying issue is that the `PageView` used by `TabBarView` is not accessible.
